content/references/http-websocket-apis/public-api-methods/ledger-methods/ledger_data.md

```diff
@@ -50,13 +50,14 @@ An example of the request format:
 
 A request can include the following fields:
 
-| `Field`        | Type                      | Required? | Description    |
-|:---------------|:--------------------------|:----------|----------------|
-| `ledger_hash`  | String - [Hash][]         | No        | A 20-byte hex string identifying the ledger version to use. |
-| `ledger_index` | Number - [Ledger Index][] | No        | The [ledger index][] of the ledger to use, or a shortcut string to choose a ledger automatically. (See [Specifying Ledgers][]) |
-| `binary`       | Boolean                   | No        | If `true`, return ledger entries as hexadecimal strings instead of JSON. The default is `false`. |
-| `limit`        | Integer                   | No        | Limit the number of ledger entries to retrieve. The server may return fewer than this number of entries. Cannot be more than 2048 (when requesting binary) or 256 (when requesting JSON). The default is the maximum. |
-| `marker`       | [Marker][]                | No        | Value from a previous paginated response. Resume retrieving data where that response left off. |
+| `Field`        | Type             | Required? | Description    |
+|:---------------|:-----------------|:----------|----------------|
+| `ledger_hash`  | [Hash][]         | No        | A 20-byte hex string identifying the ledger version to use. |
+| `ledger_index` | [Ledger Index][] | No        | The [ledger index][] of the ledger to use, or a shortcut string to choose a ledger automatically. (See [Specifying Ledgers][]) |
+| `binary`       | Boolean          | No        | If `true`, return ledger entries as hexadecimal strings instead of JSON. The default is `false`. |
+| `limit`        | Number           | No        | Limit the number of ledger entries to retrieve. The server may return fewer than this number of entries. Cannot be more than 2048 (when requesting binary) or 256 (when requesting JSON). The default is the maximum. |
```

Comment:
In the past we've had people (more than one!) who were confused that the response did not include _exactly_ the specified `limit` especially if there are more items to fetch on the next page. You might think "maximum" or "limit" makes it clear this is not guaranteed, but turns out, people expect it to return exactly the limit if at all possible—probably because other stuff like SQL queries typically work that way. So I'm leaving this in here and elsewhere.